contracts/interfaces/INonceManager.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.12;
+
+interface INonceManager {
+
+    /**
+     * return the next nonce for this sender
+     * @param sender the account address
+     * @param key the high 192 bit of the nonce
+     * @return nonce a full nonce to pass for next UserOp with this sender.
+     */
+    function getNonce(address sender, uint192 key)
+    external view returns (uint256 nonce);
+
+    function incrementNonce(uint192 key) external;
```

Comment:
Maybe add documentation, explaining that this function doesn't need to be called by the account?

Reply:
I thought it was needed for the "bypass" case where an account wants to support non-EntryPoint interactions and still use the same nonce.  Is that no longer the case?